### DISCUSSION TITLE
Asian characters not displayed in gogglesmm

### DISCUSSION THREAD

**Discussion by jarbowski** [gogglesmm/gogglesmm#34] [Q&A] [answered]
Asian characters are well displayed on my system (pcmanfm on archlinux).

**Comment by nerijus** [comment]
Can you give an example file?

**Comment by jarbowski** [comment]
![2015-09-29](https://cloud.githubusercontent.com/assets/6986038/10166304/abbe08b8-66c3-11e5-91fa-72a5b67f53cd.jpg)

**Comment by nerijus** [comment]
I can confirm - I renamed some mp3 file to include Chinese characters, cleared id3 tag, and when browsing file system gogglesmm displays empty boxes instead of Chinese chars.

**Comment by gogglesguy** [comment, marked as answer]
You'll have to switch the font in the gogglesmm preferences dialog to one that includes asian characters. It's a limitation of FOX that it can only display text in one font at a time.
